jobclient/java/src/main/java/com/twosigma/cook/jobclient/Group.java

```diff
@@ -79,6 +69,7 @@ public static Status fromString(String name) {
         private HostPlacement _hostPlacement;
         private StragglerHandling _stragglerHandling;
         private List<UUID> _jobs = new ArrayList<>();
+        private boolean _shouldValidate = true;
```

Comment:
there's no validation in cook scheduler, so let's remove validation from the client

Reply:
Sounds good will do

Reply:
Done